qlib/task_mgr.rs

```diff
@@ -64,21 +89,19 @@ pub struct Links {
 #[derive(Debug)]
 #[repr(C)]
 pub struct Context {
-    pub rsp: u64,
-    pub r15: u64,
-    pub r14: u64,
-    pub r13: u64,
-    pub r12: u64,
-    pub rbx: u64,
-    pub rbp: u64,
-    pub rdi: u64,
-
-    pub ready: AtomicU64,
-    pub fs: u64,
-    pub savefpsate: bool,
+    pub rsp: u64,  // 0x00
+    pub r15: u64,  // 0x08
+    pub r14: u64,  // 0x10
+    pub r13: u64,  // 0x18
+    pub r12: u64,  // 0x20
+    pub rbx: u64,  // 0x28
+    pub rbp: u64,  // 0x30
+    pub rdi: u64,  // 0x38
+
+    pub fs: u64,   // 0x40
+    pub savefpsate: bool,  // 
     pub X86fpstate: Option<Box<X86fpstate>>,
-    // job queue id
-    pub queueId: AtomicUsize,
+
```

Comment:
are you sure this change to context struct definition is consistent across the codebase?

Reply:
This change should only be used in CC, I guess. Other modes should use the original context.  Will add the cfg flag very soon